src/lightning-stubs/messageService/messageService.js

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+export const APPLICATION_SCOPE = jest.fn();
```

Comment:
It's fine to add it now, but I still think as a library, it should be mocked by who use library in jest. 

Is there any request for this to be stubs?

If not I would suggest we ignore this now. Until we have more use case to ask for it.

Reply:
@trevor-bliss could we just update support for component?

Reply:
This package provides default stubs for all the lightning base components that are available for customers to use. The functionality is minimal in the stubs, but it allows users to test their custom components without any further modifications most of the time. 

Without the default stubs users would have to create their own mock for every base component or library they used and they would inevitably get stale over time. Users can still easily override the default with their own custom mock whenever they want, they'll just be on the hook for keeping it up to date with the real component themselves. Here's an example of how lwc-recipes does that: 
https://github.com/trailheadapps/lwc-recipes/blob/master/jest.config.js#L6

For this specific component, someone did happen to file an issue for support: https://github.com/salesforce/sfdx-lwc-jest/issues/144

I'm not sure what you mean by updating support for the component. Could you expand on that?

Reply:
@albert-sf So are we good to merge this one? Or did you want to discuss further?

Reply:
I'm OK with it now. But long term, since stubs is majorly to help component snapshot, not library.

#144 won't make sense if they want to test to use library for behavior. it could lead to more issue.